ParseLiveQuery/src/main/java/com/parse/ParseLiveQueryClientImpl.java

```diff
@@ -341,11 +385,13 @@ public Void then(Task<Void> task) {
             @Override
             public void onClose() {
                 Log.v(LOG_TAG, "Socket onClose");
+                dispatchDisconnected();
```

Comment:
dispatchConnected() happens only when the server responds with the `op=connected` message